### DISCUSSION TITLE
how to toggle esp?

### DISCUSSION THREAD

**Discussion by ivjvon** [winstxnhdw/lc-hax#271] [Q&A] [answered]
is there a command or a bind to turn on or off the esp? I want to include the grab far mod.

**Comment by ivjvon** [comment]
How do I also turn off the interactions through walls?

**Reply by winstxnhdw** [reply]
Delete this [file](https://github.com/winstxnhdw/lc-hax/blob/main/lc-hax/Scripts/Patches/InfiniteGrabPatch.cs).

**Comment by winstxnhdw** [comment]
ESP is always on.

**Comment by winstxnhdw** [comment, marked as answer]
You can now toggle ESP  with the <kbd> Pause </kbd> key.

**Reply by ivjvon** [reply]
How do you update it? I deleted esp patch and in the loader.cs

**Reply by winstxnhdw** [reply]
You can just `git reset —hard`
